src/js/components/modals/ModalHeading.js

```diff
@@ -2,25 +2,19 @@ import classNames from "classnames";
 import React from "react";
 
 const ModalHeading = props => {
-  const { align, children, className, flush, level } = props;
+  const { children, level } = props;
```

Comment:
We now have props in the propTypes and default props that are not used within the component, let's remove them 💪

Reply:
fixed 👍